commands/create_test.go

```diff
@@ -20,3 +21,19 @@ func TestValidateSwarmDiscoveryAcceptsValidFormat(t *testing.T) {
 	err := validateSwarmDiscovery("token://deadbeefcafe")
 	assert.NoError(t, err)
 }
+
+func TestRejectCreateTrailingOptions(t *testing.T) {
```

Comment:
I strongly disagree @nathanleclaire. We should test most error cases without integration tests. This test might me a little complicated for now but it's not an integration test. It's just that the design has to be improved to make testing easier.

Until we do those improvements, that's one more test on a file that lacks tests